### PR TITLE
pushing initial testpage into main- collab purposes

### PR DESCRIPTION
**Pull request by jyyyyylim**
path: `projectroot/app/dev/index.html`

### CODE REVIEW

File: app/dev/index.html [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+    <div class="div">dick</div>
+
```
Comment (JacksonTai) [inline app/dev/index.html:245]: What's this?

Comment (jyyyyylim) [reply app/dev/index.html:245]: recreation of index.php from prototype
so far i did halfway only, but merge in first so others can continue when they want

at least could faster finish

Comment (jyyyyylim) [reply app/dev/index.html:245]: divs spammed to test scrolling bc of sticky navbar, will be removed when more content filled in. probably tonight

Comment (JacksonTai) [reply app/dev/index.html:245]: Ok, I'm also working on the main page, probably will do a pr once I got the structure

Comment (jyyyyylim) [reply app/dev/index.html:245]: which part you working?

Comment (JacksonTai) [reply app/dev/index.html:245]: **I'm working on:** 
- home & signin & signup pages (covering both HTML and CSS)

**I just finished:**
- layout files such as header and footer.
- a global CSS file that will be included in all pages.

If you guys have already did or going to do some of these works, just go ahead, cause I will be finishing these on today.

**Pull request closed without merging**